src/common-types.ts

```diff
@@ -133,3 +133,17 @@ export interface TooltipProps {
 export interface Tooltip {
 	tooltip?: string | TooltipProps;
 }
+
+export const orderedActionTypes: string[] = [
+	'none',
+	'primary',
+	'warning',
```

Comment:
I really don't think we need a warning action, it's either dangerous or it isn't

Reply:
I was thinking that maybe at some point we might need something like: "this action requires a restart" which is not dangerous but warns the user that there will be a consequence. For now, however, I take it off, then we can add it back in case.

Reply:
yeah let's do that. It just seems weird figuring out what s dangerous, normal, or a warning, and it's simpler if we just have dangerous/not dangerous (as almost any other app really)